packages/common-db/src/lib/common/session.ts

```diff
@@ -1,24 +1,33 @@
-type Session = any;
+type SessionData = Record<string, any>;
+
+type Session = {
+    Store: any;
+};
 
 // TODO: in the long term move this file somewhere where we have types access it is nowhere used in controller itself and just exported for adapters so it should go to js-controller-adapter package
 interface AdapterStoreOptions {
     /** The ioBroker adapter */
     adapter: any;
```

Comment:
why Any and not Adapter type?

Reply:
![image](https://github.com/user-attachments/assets/2577b080-e6e7-4705-90ac-4cd978b328b8)
IDK how to solve it

Reply:
The types are not build when DB is build so can not be used. So maybe move AdapterStore out of Common-DB package when it is more Adapter specific?

Reply:
https://github.com/ioBroker/ioBroker.js-controller/issues/3045